test/integration/execute.spec.ts

```diff
@@ -692,6 +698,42 @@ describe('.execute()', () => {
 		);
 		expect(threadResult.error).toBe(false);
 
+		// Before enqueing up a create event we need to create a relationship to the random-slug type created above
+		const relationship: RelationshipContractDefinition = {
+			slug: `relationship-card-is-attached-to-${slug}`,
+			type: 'relationship@1.0.0',
+			name: 'is attached to',
+			data: {
+				inverseName: 'has attached element',
+				title: _.capitalize(slug),
+				inverseTitle: 'Card',
+				from: {
+					type: 'card@1.0.0',
+				},
+				to: {
+					type: `${slug}@1.0.0`,
+				},
+			},
+		};
+		const relationshipTypeContract =
+			await ctx.kernel.getContractBySlug<TypeContract>(
+				ctx.logContext,
+				ctx.session,
+				'relationship@1.0.0',
+			);
+		if (!relationshipTypeContract) {
+			throw new errors.WorkerNoElement(`No relationship type contract found`);
+		}
```

Comment:
Can you not pull this from the worker instance? e.g. `ctx.worker.typeContracts['relationship@1.0.0']`